### PR TITLE
CHANGELOG Translation

### PR DESCRIPTION
**Pull request by sushi2k**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [ ] Your PR is linked to an issue.
- [ ] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #< insert number here >.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

File: Document-ko/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## V1.2 - 2020년 3월 7일 - 국제 릴리스
 
+The following changes are part of release 1.2:
```
Comment (OWASP-Seoul) [inline Document-ko/CHANGELOG.md]: 다음 변경 사항은 릴리스 1.2의 일부입니다:

File: Document-ko/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
...
+
+- Translation in simplified Chinese of the MASVS available.
```
Comment (OWASP-Seoul) [inline Document-ko/CHANGELOG.md]: - MASVS의 중국어 간체 번역.

File: Document-ko/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
...
+- Change of title in MASVS book cover.
```
Comment (OWASP-Seoul) [inline Document-ko/CHANGELOG.md]: - MASVS 책 표지에서 제목 변경.

File: Document-ko/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,12 @@
...
+- Removed Mobile Top 10 and CWE from MSTG and merged to existing references in MASVS.
```
Comment (OWASP-Seoul) [inline Document-ko/CHANGELOG.md]: - MSTG에서 Mobile Top 10 및 CWE를 제거하고 MASVS의 기존 참조와 병합.

File: Document-ko/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,12 @@
...
+
 ## V1.2-RC - 2019년 10월 5일 Pre-release
 
 다음 변경 사항은 릴리스 1.2의 일부입니다:
```
Comment (OWASP-Seoul) [inline Document-ko/CHANGELOG.md]: 다음 변경 사항은 시험판 1.2의 일부입니다:

File: Document-ko/CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,12 @@
 
 ## V1.2 - 2020년 3월 7일 - 국제 릴리스
 
+The following changes are part of release 1.2:
+
+- Translation in simplified Chinese of the MASVS available.
+- Change of title in MASVS book cover.
+- Removed Mobile Top 10 and CWE from MSTG and merged to existing references in MASVS.
+
 ## V1.2-RC - 2019년 10월 5일 Pre-release
```
Comment (OWASP-Seoul) [inline Document-ko/CHANGELOG.md]: ## V1.2-RC - 2019년 10월 5일 - 시험판

File: Document-es/CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,12 @@
 
 ## V1.2 - 7 de marzo 2020 - Release Internacional
 
+The following changes are part of release 1.2:
```
Comment (cpholguera) [inline Document-es/CHANGELOG.md]:
```suggestion
Los siguientes cambios forman parte de la versión 1.2:
```

File: Document-es/CHANGELOG.md [hunk 8]
```diff
@@ -2,6 +2,12 @@
...
+
+- Translation in simplified Chinese of the MASVS available.
```
Comment (cpholguera) [inline Document-es/CHANGELOG.md]:
```suggestion
- El MASVS ha sido traducido al chino simplificado.
```

File: Document-es/CHANGELOG.md [hunk 9]
```diff
@@ -2,6 +2,12 @@
...
+- Change of title in MASVS book cover.
```
Comment (cpholguera) [inline Document-es/CHANGELOG.md]:
```suggestion
- El título ha sido cambiado en la portada del MASVS.
```

File: Document-es/CHANGELOG.md [hunk 10]
```diff
@@ -2,6 +2,12 @@
...
+- Removed Mobile Top 10 and CWE from MSTG and merged to existing references in MASVS.
```
Comment (cpholguera) [inline Document-es/CHANGELOG.md]:
```suggestion
- Todas las referencias CWE y Mobile Top 10 han sido eliminadas de la MSTG y han sido integradas a las referencias existentes en el MASVS.
```

File: Document-ru/CHANGELOG.md [hunk 11]
```diff
@@ -2,6 +2,12 @@
 
 ## V1.2 - 7 марта 2020 - Международный релиз
 
+The following changes are part of release 1.2:
+
+- Translation in simplified Chinese of the MASVS available.
```
Comment (emartynov) [inline Document-ru/CHANGELOG.md]:
```suggestion
- MASVS доступен на упрощенном китайском.
```

File: Document-ru/CHANGELOG.md [hunk 12]
```diff
@@ -2,6 +2,12 @@
...
+- Change of title in MASVS book cover.
```
Comment (emartynov) [inline Document-ru/CHANGELOG.md]:
```suggestion
- Изменено название в обложке книги MASVS.
```

File: Document-ru/CHANGELOG.md [hunk 13]
```diff
@@ -2,6 +2,12 @@
...
+- Removed Mobile Top 10 and CWE from MSTG and merged to existing references in MASVS.
```
Comment (emartynov) [inline Document-ru/CHANGELOG.md]:
```suggestion
- Удалены Mobile Top 10 и CWE из MSTG и объединены с существующими ссылками в MASVS.
```

**Comment by ninedter** [comment]
approve to TW version.

File: Document-fr/CHANGELOG.md [hunk 14]
```diff
@@ -2,6 +2,12 @@
 
 ## V1.2 - 7 Mars 2020 - Edition Internationale
 
+The following changes are part of release 1.2:
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: Les changements suivants ont été intégrés dans la version 1.2 :

File: Document-fr/CHANGELOG.md [hunk 15]
```diff
@@ -2,6 +2,12 @@
...
+
+- Translation in simplified Chinese of the MASVS available.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - Traduction du MASVS  en chinois simplifié disponible.

File: Document-fr/CHANGELOG.md [hunk 16]
```diff
@@ -2,6 +2,12 @@
...
+- Change of title in MASVS book cover.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - Changement du titre en couverture du livre sur le MASVS.

File: Document-fr/CHANGELOG.md [hunk 17]
```diff
@@ -2,6 +2,12 @@
...
+- Removed Mobile Top 10 and CWE from MSTG and merged to existing references in MASVS.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - Suppression du Top 10 Mobile et de CWE du MSTG et fusion avec les références existantes dans le MASVS.

File: Document-fr/CHANGELOG.md (hunk 14 above)
Comment (commjoen) [reply Document-fr/CHANGELOG.md]:
```suggestion
Les changements suivants ont été intégrés dans la version 1.2 :
```

File: Document-fr/CHANGELOG.md (hunk 15 above)
Comment (commjoen) [reply Document-fr/CHANGELOG.md]:
```suggestion
- Traduction du MASVS en chinois simplifié disponible.
```

File: Document-fr/CHANGELOG.md (hunk 16 above)
Comment (commjoen) [reply Document-fr/CHANGELOG.md]:
```suggestion
- Change of title in MASVS book cover.
```

File: Document-fr/CHANGELOG.md (hunk 17 above)
Comment (commjoen) [reply Document-fr/CHANGELOG.md]:
```suggestion
- Suppression du Top 10 Mobile et de CWE du MSTG et fusion avec les références existantes dans le MASVS.
```

**Comment by commjoen** [comment]
Note , we should not forget to put "powered by pandocer" somewhere in our README: https://github.com/dalibo/pandocker/issues/145#issuecomment-581303361

**Comment by sushi2k** [comment]
> Note , we should not forget to put "powered by pandocer" somewhere in our README: [dalibo/pandocker#145 (comment)](https://github.com/dalibo/pandocker/issues/145#issuecomment-581303361)

Thanks Jeroen. Just added

**Pull request merged**